lib/rialto/etl/configs/sparql.rb

```diff
@@ -2,6 +2,11 @@
 
 require 'rialto/etl/readers/sparql_statement_reader'
 require 'rialto/etl/writers/sparql_writer'
+
+extend Rialto::Etl::Logging
```

Comment:
Does this need a `require` statement above?

Reply:
Added.